docs/releases/upgrading/upgrade-v12.0.md

```diff
@@ -0,0 +1,215 @@
+# Upgrading from v11 to v12
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v11.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older Versions"
+
+    In case you have an InvenioRDM installation older than v11, you can gradually upgrade
+    using the existing infrastructure to v11 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli` installed. For InvenioRDM v12 it
+    is at least v1.2.0
+
+    ```bash
+    $ invenio-cli --version
+    invenio-cli, version 1.2.0
+    ```
+
+!!! info "Virtual environments"
+
+    In case you are not inside a virtual environment, make sure that you prefix each `invenio`
+    command with `pipenv run`.
+
+
+**Local development**
+
+Changing the Python version in your development environment highly
```

Comment:
I think this is leftover. If folks are on v11, they should already have python 3.9 and won't need to change.

Reply:
if they are then they can skip that step, if they are not than this is a really necessary step. i would keep it.

Reply:
I would then start it with something like. "Python 3.9 or Python 3.12 are required to run InvenioRDM."